docs/src/manual.md

```diff
@@ -6,7 +6,7 @@ using JuMP
 using ToQUBO
 using Anneal
 
-model = Model(() -> ToQUBO.Optimizer(SimulatedAnnealer.Optimizer))
+model = Model(() -> ToQUBO.Optimizer(RandomSampler.Optimizer))
 
```

Comment:
Why changing this to random sampling? If anything, do also total enumeration and use the manual to warn against this.

Reply:
The `SimulatedAnnealer` is undefined (now the separate package `DWaveNeal`). I have just set it to `RandomSampler` for it to run without errors.

Reply:
I updated it to use `DWaveNeal`.